flake.nix

```diff
@@ -560,14 +554,9 @@
               installPhase = ''
                 mkdir $out
                 nixfmt --version
-
-                total_exit_code=0
-                for file in $(find ${all-directories-and-files} -type f -and -name "*.nix"); do
-                  echo "=== $file ==="
-                  nixfmt --check $file || total_exit_code=$?
-                  echo "==="
-                done
-                exit $total_exit_code
+                # note, really can just src with filer by .nix, no need all files 
```

Comment:
yes, but ` ${all-directories-and-files}` is more reusable
e.g. [remark check](https://github.com/ComposableFi/composable/blob/main/.github/workflows/check.yml#L142-L150) can reuse it

Reply:
same applies, you do src = ./. where name is *.md and type is regular

Reply:
so check will rebuilt iff only md changes